bot/config.py.zephyr.sample

```diff
@@ -99,7 +99,7 @@ z['iut_config'] = {
     },
 
     "privacy.conf": {
-        "overlay": {'CONFIG_BT_PRIVACY': 'y'},
+        "overlay": {'CONFIG_BT_PRIVACY': 'y', 'CONFIG_BT_RPA_TIMEOUT': '5'},
```

Comment:
@SondreRavn did you investigate the effect of adding CONFIG_BT_RPA_TIMEOUT on the other tests in the privacy.conf?  If yes we, I will approve